### ISSUE TITLE
Plugin: NWNX_Events NWNX_ON_RESOURCE_ADDED

### ISSUE THREAD

**Issue by Morderon** [nwnxee/unified#1049]
NWNX_ON_RESOURCE_ADDED (and likely the others) don't fire for resources in the development folder

D [20:40:25] [NWNX_Events] [ResourceEvents.cpp:56] Adding path '/home/md/.local/share/Neverwinter Nights/nwnx/' for alias 'NWNX'
D [20:40:25] [NWNX_Events] [ResourceEvents.cpp:56] Adding path '/home/md/.local/share/Neverwinter Nights/development/' for alias 'DEVELOPMENT'
I [20:40:25] [NWNX_Events] [Events.cpp:252] Script 'evt_res_add' subscribed to event 'NWNX_ON_RESOURCE_ADDED'.

-

The resource did update when placed in the development folder.
No event data was pushed and the script didn't fire.

The script did fire correctly when the resource was copied to the nwnx folder.

**Comment by Daztek** [comment]
It seems to working for me.

```
D [09:05:39] File 'fnt_map.tga' was 'ADDED' in alias 'DEVELOPMENT'
D [09:05:39] Pushing event data: 'ALIAS' -> 'DEVELOPMENT'.
D [09:05:39] Pushing event data: 'RESREF' -> 'fnt_map'.
D [09:05:39] Pushing event data: 'TYPE' -> '3'.
D [09:05:39] Dispatching notification for event 'NWNX_ON_RESOURCE_ADDED' to script 'es_s_testing'.```

**Comment by Morderon** [comment]
Okay additional details after testing

1) Linux is a virtual machine on my test system and the development folder is shared across host and the VM. Placing the file into the hosts development folder doesn't trigger the event but placing it in on the Linux side does.

2) It seems the add/remove/modify doesn't trigger if you just change the file's extension or file's name

3) If I removed a file and readded it, modified also triggered where I would had assumed only added would had.

All and all pretty small stuff now that I know about it.

**Comment by Daztek** [comment]
There's some weird quirks yeah. I use it to watch for `.nss` file changes, and if the file is bigger than 4096 bytes it'll trigger the modified event twice, for ??? reasons.

**Comment by mtijanic** [comment]
Can we close this one, per #1050 ?

**Comment by mtijanic** [comment]
Closing as fixed in #1050 since no one claimed otherwise :)
